src/pudl/metadata/resources/eia861.py

```diff
@@ -49,338 +65,553 @@
+                'load_management_annual_effects_mwh',
+                'load_management_annual_incentive_payment',
+                'load_management_annual_potential_peak_reduction_mw',
+                'load_management_incremental_actual_peak_reduction_mw',
+                'load_management_incremental_effects_mwh',
+                'load_management_incremental_potential_peak_reduction_mw',
+                'nerc_region',
+                'price_responsiveness_customers',
+                'report_date',
+                'state',
+                'time_responsiveness_customers',
+                'utility_id_eia'
+            ],
+        },
+        "group": "eia",
+        "sources": ["eia861"],
+    },
+    "demand_side_management_misc_eia861": {
+        "description": "",
+        "schema": {
+            "fields": [
+                'energy_savings_estimates_independently_verified',
+                'energy_savings_independently_verified',
+                'entity_type',
+                'major_program_changes',
+                'nerc_region',
+                'price_responsive_programs',
+                'report_date',
+                'reported_as_another_company',
+                'short_form',
+                'state',
+                'time_responsive_programs',
+                'utility_id_eia',
+                'utility_name_eia'
+            ],
+        },
+        "group": "eia",
+        "sources": ["eia861"],
+    },
+    "demand_side_management_sales_eia861": {
+        "description": "",
+        "schema": {
+            "fields": [
+                'nerc_region',
+                'report_date',
+                'sales_for_resale_mwh',
+                'sales_to_ultimate_consumers_mwh',
+                'state',
+                'utility_id_eia'
+            ],
+        },
+        "group": "eia",
+        "sources": ["eia861"],
+    },
+    "distributed_generation_fuel_eia861": {
+        "description": "",
+        "schema": {
+            "fields": [
+                'estimated_or_actual_fuel_data',
+                'fuel_class',
+                'fuel_pct',
+                'report_date',
+                'state',
+                'utility_id_eia'
+            ],
+        },
+        "group": "eia",
+        "sources": ["eia861"],
+    },
+    "distributed_generation_misc_eia861": {
+        "description": "",
+        "schema": {
+            "fields": [
+                'backup_capacity_mw',
+                'distributed_generation_owned_capacity_mw',
+                'estimated_or_actual_capacity_data',
+                'generators_num_less_1_mw',
+                'generators_number',
+                'report_date',
+                'state',
+                'total_capacity_less_1_mw',
+                'utility_id_eia',
+                'utility_name_eia'
+            ],
+        },
+        "group": "eia",
+        "sources": ["eia861"],
+    },
+    "distributed_generation_tech_eia861": {
+        "description": "",
```

Comment:
Should we add descriptions to these new schemas and fields?

Reply:
Yes!

Reply:
Some of them were straightforward because they come directly from an EIA table, but others require some more research, since we're splitting apart the spreadsheets into normalized tables, and I wasn't sure off the top of my head what the right way to describe the differences were.

Reply:
Also, there's going to have to be another fairly deep dive into the EIA-861 metadata to get all this working with the database -- and these resource definitions are currently getting skipped in the compilation of the `RESOURCE_METADATA` so I was kind of deferring this work until later, while still trying to get a first pass here.

Reply:
Sounds good 👍